### PR TITLE
Upgrade from k8s v1.23.8 to v1.25.x in two steps

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by lentzi90 — COMMENTED** [review]
/lgtm

File: tests/roles/run_tests/tasks/upgrade_k8s.yml [hunk 1]
```diff
@@ -0,0 +1,212 @@
+    - name: Download upgraded boot-disk image for deployment
```
Comment (furkatgofurov7) [inline tests/roles/run_tests/tasks/upgrade_k8s.yml:1]: This file's indentation, shift to right a bit more compared to other files?

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**